encryption_works.md

```diff
@@ -98,25 +98,41 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
 
-Even if they can, using Tor still gives us many advantages. It makes their job much harder, and we leave much less identifying data on the servers we connect to through the Tor network. It makes it much harder to be the victim of a MITM attack at our local network or ISP level. And even if some Tor circuits can be defeated by a global adversary, if enough people are getting their traffic routed through the same Tor nodes at the same time, it might be difficult for the adversary to tell which traffic belongs to which circuits.
+Recently, the Tor Project [commented](https://blog.torproject.org/blog/preliminary-analysis-hacking-teams-slides) on reports that Hacking Team, the surveillance technology vendor which was massively hacked in July 2015, had managed to compromise the Tor network. The attack turned out to be a case of targeted surveillance rather than a widespread problem:
 
-The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en).
+> The good news is that they don't appear to have any exploit on Tor or on Tor Browser. The other good news is that their proposed attack doesn't scale well. They need to put malicious hardware on the local network of their target user, which requires choosing their target, locating her, and then arranging for the hardware to arrive in the right place. So it's not really practical to launch the attack on many Tor users at once.
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/torbrowser.png)
+As with all privacy-enhancing software, an attacker with physical access to the device can compromise you in any number of ways.
 
-When Snowden [was answering questions on Guardian's website](http://www.guardian.co.uk/world/2013/jun/17/edward-snowden-nsa-files-whistleblower) from a "secure Internet connection", he was probably routing his traffic through the Tor network. He may have also been using a [bridge](https://bridges.torproject.org/) to connect to the Tor network to make the fact that he was using Tor from his IP address less obvious to eavesdroppers.
+### Setup
+
+The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en). 
+
+### Browsing Habits
+
+You should use Tor differently than you use a regular browser: some of the features it uses to obscure your location can be undermined if you don't take a few extra steps to protect yourself.
```

Comment:
I would change this to something like "Here are some further tips to enhance your security and privacy while using the Tor browser (or when using any web browser." Arguably people should turn off flash and do these things in their regular browser too.